### PR TITLE
Add Conctact form

### PR DESCRIPTION
**Pull request by benja27**
Add contact form to the main branch
Use Formspree service to validate the form

### CODE REVIEW

**Review by shella12 — COMMENTED** [review]
Hi @benja27 , 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that

- [ ] Does not have descriptive PR, Please make sure to add  descriptive summary and a meaningful title to you PR indicating the changes you have made 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by benja27** [comment]
Add contact form to the main branch
Use Formspree service to validate the form

File: index.html [hunk 1]
```diff
@@ -229,10 +229,10 @@ <h3 class="hero-text hero-text-white">
...
-            <form action="" class="contact-form">
-                <input type="text" name="" id="" placeholder="Yeremias" >
-                <input type="email" name="" id="" placeholder="natajayanj14@gmail.com" >
-                <textarea name="" id="" cols="30" rows="10" placeholder="Write your message here" ></textarea>
+            <form action="https://formspree.io/f/xjvdwrqb" method="POST" class="contact-form">
+                <input required type="text" name=""  id="" placeholder="Ingresa tu nombre" maxlength="30" >
+                <input required type="email" name="" id="" placeholder="Ingresa tu correo" >
+                <textarea name="coment" cols="30" rows="10" placeholder="Ingresa tus comentarios" maxlength="500"></textarea>
                 <input class="btn" type="submit" value="Get in touch">
             </form>
```
Comment (AdnanOlarmmi) [inline index.html:237]: - [ ] Kindy ensure to remove the default input border, on enable and not on focus, either by making the `outline` property zero or by setting the `border` color to transparent. 😃 

- Your design,
wrong

- Figma design,
right

File: index.html [hunk 2]
```diff
@@ -229,10 +229,10 @@ <h3 class="hero-text hero-text-white">
...
             </h3>
 
-            <form action="" class="contact-form">
-                <input type="text" name="" id="" placeholder="Yeremias" >
-                <input type="email" name="" id="" placeholder="natajayanj14@gmail.com" >
-                <textarea name="" id="" cols="30" rows="10" placeholder="Write your message here" ></textarea>
+            <form action="https://formspree.io/f/xjvdwrqb" method="POST" class="contact-form">
+                <input required type="text" name=""  id="" placeholder="Ingresa tu nombre" maxlength="30" >
+                <input required type="email" name="" id="" placeholder="Ingresa tu correo" >
+                <textarea name="coment" cols="30" rows="10" placeholder="Ingresa tus comentarios" maxlength="500"></textarea>
```
Comment (AdnanOlarmmi) [inline index.html:235]: Please, kindly ensure that the `textarea` is also `required` as specified in the requirements 😉.

File: index.html [hunk 3]
```diff
@@ -229,10 +229,10 @@ <h3 class="hero-text hero-text-white">
...
                 <input class="btn" type="submit" value="Get in touch">
```
Comment (AdnanOlarmmi) [inline index.html:236]: 😀 From what I noticed here, it seems the submit button has a little `box-shadow` design **on hover** from the template, if this is true, kindly implement it as such 😉.

- Your design
wrong


- Figma design
right

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @benja27, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- Professional Readme.
- Nice UI design.

## Required Changes ♻️
- [ ] Kindly ensure that your partner collaborates with you on this task, I should have invalidated your PR, but the first invalidation reason misled me and made me forget that totally. Thanks 😀. Kindly pardon me.
_Check the comments under the review for more._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (benja27) [reply index.html:236]: image

the box shadow is already there

**Review by topeogunleye — APPROVED** [review]
### Hi @benja27, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by benja27** [comment]
Not nesesary

**Pull request closed without merging**